source/lac/petsc_matrix_base.cc

```diff
@@ -331,6 +331,35 @@ namespace PETScWrappers
 
 
 
+  MatrixBase::size_type
+  MatrixBase::local_domain_size() const
+  {
+    PetscInt n_cols;
+
+    const PetscErrorCode ierr = MatGetLocalSize(matrix, nullptr, &n_cols);
+    AssertThrow(ierr == 0, ExcPETScError(ierr));
+
+    return n_cols;
+  }
+
+
+
+  std::pair<MatrixBase::size_type, MatrixBase::size_type>
+  MatrixBase::local_domain() const
+  {
+    PetscInt begin, end;
+
+    const PetscErrorCode ierr =
+      MatGetOwnershipRangeColumn(static_cast<const Mat &>(matrix),
+                                 &begin,
+                                 &end);
+    AssertThrow(ierr == 0, ExcPETScError(ierr));
+
+    return std::make_pair(begin, end);
```

Comment:
```suggestion
    return {begin, end};
```

Reply:
this was copy-pasted from the row method. Do you want me to apply the same suggestion also there?

Reply:
done